blog/26-08-2022-creating-a-hook-for-the-go-sdk.mdx

```diff
@@ -0,0 +1,142 @@
+---
+slug: creating-a-hook-for-the-go-sdk
+title: Creating a Hook for the golang-sdk
+authors: [skyerus]
+tags: [golang, blog, tutorial, golang-sdk, hooks]
+unlisted: false
+---
+
+## Hooks
+
+A [Hook](https://docs.openfeature.dev/docs/specification/sections/hooks) taps into one or more of the flag evaluation's
+lifecycle events (before/after/error/finally) to perform the same action at that point for every evaluation.
+
+## Objective
+
+<i>Create and integrate a spec compliant hook that validates that the result of a flag evaluation is a hex color.</i>
+
+## Prerequisites
+
+- Golang 1.17
```

Comment:
Is the min version 1.17 or is it the required version?

Reply:
Ya, I think this should say `Golang 1.17+`

Reply:
Nice spot, done!